omnibus/Gemfile

```diff
@@ -18,6 +18,3 @@ group :development do
   gem "kitchen-vagrant"
   gem "winrm-elevated"
 end
-
-# TODO remove this when we update Chef to use the new api exposed in 3.5.1
-gem "rspec-core", "= 3.4.4"
```

Comment:
???

Reply:
Yeah. Apparently I added that 2 years ago in https://github.com/chef/chef-dk/commit/d11e56352b4fefdd052c0808942e104d8d8bdd4f - I see no reason to keep it pinned in the omnibus build since it is no longer pinned in the main Gemfile

Reply:
Is our rspec installation used directly by customers, or do we only build it for self-testing?